app/views/admin/reports/show.html.erb

```diff
@@ -21,6 +21,19 @@
 <h3>Students</h3>
 <p>
   Student accounts created since July 1: <%= User.student.where(created_at: DateTime.beginning_of_rice_fiscal_year..DateTime.now).count %> <br>
-  Student accounts created in the past week: <%= User.student.where(created_at: DateTime.now - 1.week..DateTime.now).count %>
-</p>
+  Student accounts created in the past week: <%= User.student.where(created_at: DateTime.now - 1.week..DateTime.now).count %> <br>
+  How many people start the account creation process versus how many finish it? <br>
+  <% since_year = 2018 %>
+  <% years = since_year..Time.now.year %>
+  <% years.each do |year| %>
+    <% time_range = Time.local(year)..Time.local(year + 1) %>
+    <% pas = PreAuthState.where(created_at: time_range).count %>
+    <% users = User.where(state: User::ACTIVATED, created_at: time_range).count %>
```

Comment:
this should live in a model, as a scope, where it can be unit tested and reused.  the whole years stuff here encapsulated in a helper